server/utilities.js

```diff
@@ -1,51 +1,22 @@
-var Promise = require('bluebird');
-var queries = require('./queries.js');
-var memoBuild = memoize(buildFunc);
+var memoize = module.exports.memoize = function(func) {
```

Comment:
do you need to say both `var memoize` and `module.exports.memoize`? If you can just say `module.exports.memoize` as you did with a few other functions below, I'd prefer we do it in that style so that we're being consistent